src/core/electric_tariff.jl

```diff
@@ -238,6 +243,22 @@ function ElectricTariff(;
         tou_demand_tier_limits = u.tou_demand_tier_limits
         n_tou_demand_tiers = u.n_tou_demand_tiers
 
+        # need to review!!!!
```

Comment:
Has this review happened? If so plz remove the comment. If not, what review is needed?